src/sql/platform/carbonEditLocalization/common/carbonEditStrings.ts

```diff
@@ -0,0 +1,74 @@
+	'update.config.contribution.enableWindowsBackgroundUpdates': nls.localize('enableWindowsBackgroundUpdates', "Enable to download and install new Azure Data Studio Versions in the background on Windows"),
+	'update.config.contribution.showReleaseNotes': nls.localize('showReleaseNotes', "Show Release Notes after an update. The Release Notes are opened in a new web browser window."),
+	'issueReporterMain.azuredatastudio': nls.localize('azuredatastudio', "Azure Data Studio"),
+	'menusExtensionPoint.dashboard.toolbar': nls.localize('dashboard.toolbar', "The dashboard toolbar action menu"),
+	'menusExtensionPoint.notebook.cellTitle': nls.localize('notebook.cellTitle', "The notebook cell title menu"),
+	'menusExtensionPoint.notebook.title': nls.localize('notebook.title', "The notebook title menu"),
+	'menusExtensionPoint.notebook.toolbar': nls.localize('notebook.toolbar', "The notebook toolbar menu"),
+	'menusExtensionPoint.dataExplorer.action': nls.localize('dataExplorer.action', "The dataexplorer view container title action menu"),
+	'menusExtensionPoint.dataExplorer.context': nls.localize('dataExplorer.context', "The dataexplorer item context menu"),
+	'menusExtensionPoint.objectExplorer.context': nls.localize('objectExplorer.context', "The object explorer item context menu"),
+	'menusExtensionPoint.connectionDialogBrowseTree.context': nls.localize('connectionDialogBrowseTree.context', "The connection dialog's browse tree context menu"),
+	'menusExtensionPoint.dataGrid.context': nls.localize('dataGrid.context', "The data grid item context menu"),
+	'extensions.contribution.extensionsPolicy': nls.localize('extensionsPolicy', "Sets the security policy for downloading extensions."),
+	'extensions.contribution.InstallVSIXAction.allowNone': nls.localize('InstallVSIXAction.allowNone', 'Your extension policy does not allow installing extensions. Please change your extension policy and try again.'),
+	'extensionsActions.postUninstallTooltip': nls.localize('postUninstallTooltip', "Please reload Azure Data Studio to complete the uninstallation of this extension."),
+	'extensionsActions.postUpdateTooltip': nls.localize('postUpdateTooltip', "Please reload Azure Data Studio to enable the updated extension."),
+	'extensionsActions.enableLocally': nls.localize('enable locally', "Please reload Azure Data Studio to enable this extension locally."),
+	'extensionsActions.postEnableTooltip': nls.localize('postEnableTooltip', "Please reload Azure Data Studio to enable this extension."),
+	'extensionsActions.postDisableTooltip': nls.localize('postDisableTooltip', "Please reload Azure Data Studio to disable this extension."),
+	'desktop.contribution.miinstallVsix': nls.localize({ key: 'miinstallVsix', comment: ['&& denotes a mnemonic'] }, "Install Extension from VSIX Package")
+};
+
+export function getCustomString(stringName: string, ...stringParams: string[]): string {
```

Comment:
Instead of a function and object separated like this it seems like it'd be a lot cleaner just to have them all defined in this file and then exported for use elsewhere like we do in extensions, see https://github.com/microsoft/azuredatastudio/blob/main/extensions/arc/src/localizedConstants.ts for an example